### ISSUE TITLE
String index out of bounds for variant GRCh37 `chr11:67258349AGCTGGACCCAGCC>GT`

### ISSUE THREAD

**Issue by ielis** [charite/jannovar#475]
Hi guys,
I am using Jannovar v0.32 to perform variant annotation and I am getting an error when I try to annotate variant `chr11:67258349AGCTGGACCCAGCC>GT`.

When I run
```bash
java -jar jannovar-cli-0.32.jar annotate-pos -d jannovar/v0.32/hg19_ensembl.ser -c 'chr11:67258349AGCTGGACCCAGCC>GT'
```

I get the following output:
```
Options
JannovarAnnotatePosOptions [genomicChanges=[chr11:67258349AGCTGGACCCAGCC>GT], toString()=JannovarAnnotationOptions [useThreeLetterAminoAcidCode=false, nt3PrimeShifting=false, showAll=false, databaseFilePath=jannovar/v0.32/hg19_ensembl.ser, toString()=JannovarBaseOptions [reportProgress=true, httpProxy=null, httpsProxy=null, ftpProxy=null, verbosity=1]]]
Deserializing transcripts...
INFO Deserializing JannovarData from jannovar/v0.32/hg19_ensembl.ser
INFO Deserialization took 8.65 sec.
#change	effect	hgvs_annotation	messages
[ERROR] Could not annotate variant chr11:67258349AGCTGGACCCAGCC>GT!
java.lang.StringIndexOutOfBoundsException: begin 229, end 233, length 232
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3319)
	at java.base/java.lang.String.substring(String.java:1874)
	at de.charite.compbio.jannovar.annotation.builders.BlockSubstitutionAnnotationBuilder$CDSExonicAnnotationBuilder.<init>(BlockSubstitutionAnnotationBuilder.java:153)
	at de.charite.compbio.jannovar.annotation.builders.BlockSubstitutionAnnotationBuilder.build(BlockSubstitutionAnnotationBuilder.java:61)
	at de.charite.compbio.jannovar.annotation.builders.AnnotationBuilderDispatcher.build(AnnotationBuilderDispatcher.java:66)
	at de.charite.compbio.jannovar.annotation.VariantAnnotator.buildNonSVAnnotation(VariantAnnotator.java:149)
	at de.charite.compbio.jannovar.annotation.VariantAnnotator.buildAnnotations(VariantAnnotator.java:130)
	at de.charite.compbio.jannovar.cmd.annotate_pos.AnnotatePositionCommand.run(AnnotatePositionCommand.java:63)
	at de.charite.compbio.jannovar.Jannovar.main(Jannovar.java:67)
```

Any idea what's going on?

Thank you,
Daniel

**Comment by visze** [comment]
I can reproduce the error.

Same problem in v.33 Only on ensemble on not in refseq

I have to take a closer look!

**Comment by visze** [comment]
This error is resolved in the actual development. Basically in the PR #472

I think we should release a new bugfix version even if ensembl mouse is not resolved yet.  But before we have to tackle bug #474 This is right now blocking for me :-(

**Comment by holtgrewe** [comment]
Hm, this is still open for me in `develop` but will be fixed by #478.
